packages/components/src/HopperProvider/tests/jest/HopperProvider.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { HopperProvider } from "../../src/index.ts";
+import { createRef } from "react";
+import { render, renderHook, screen } from "@hopper-ui/test-utils";
+import { useLocale } from "react-aria-components";
+
+describe("HopperProvider", () => {
+    it("should render with default class", () => {
+        render(
+            <HopperProvider colorScheme="light">
```

Comment:
Maybe the render with the provider can be a function instead of repeating it each time.

Reply:
Every other test does, but since we're testing the HopperProvider in this file, it's a bit awkward to have it rendered by the render wrapper.

Reply:
I just meant a little local function inside here.